src/controllers/executionPlanWebviewController.ts

```diff
@@ -0,0 +1,168 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from "vscode";
+import { ReactWebViewPanelController } from "./reactWebviewController";
+import * as ep from "../reactviews/pages/ExecutionPlan/executionPlanInterfaces";
+import { WebviewRoute } from "../sharedInterfaces/webviewRoutes";
+import * as LocalizedConstants from "../constants/localizedConstants";
+
+export class ExecutionPlanWebViewController extends ReactWebViewPanelController<
+  ep.ExecutionPlanWebViewState,
+  ep.ExecutionPlanReducers
+> {
+  constructor(
+    context: vscode.ExtensionContext,
+    private _executionPlanService: ep.ExecutionPlanService,
+    private executionPlanContents: string,
+    // @ts-ignore
+    private fileName: string

```

Comment:
how is this going to work for interactive query plans returned as XML with the query results, when there isn't a file?  Is there going to be a different controller for that?  If this controller is only for SQLPLAN files than perhaps the name of the class should reflect that.

Reply:
further in the code it looks like this is the documentTitle, so that makes sense.  Though cleaning up class and parameter names would help with code readability.